style.css

```diff
@@ -0,0 +1,412 @@
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.small-news-titles {
+  margin-top: -18px;
+  font-size: 18px;
+  width: 100%;
+  font-family: 'Croissant One';
+  color: rgba(0, 0, 0, 0.76);
+}
+
+.small-news-text {
+  margin-top: -10px;
+  width: 100%;
+  color: rgba(0, 0, 0, 0.76);
+  font-weight: bold;
+  font-family: 'IBM Plex Serif', serif;
+  font-size: 13px;
+}
+
+.read-more {
+  color: rgba(0, 0, 0, 0.76);
+}
+
+.read-more:hover {
+  color:rgb(146, 146, 112);
+}
+
+.creator {
+  font-family: 'IBM Plex Serif', serif;
+  font-weight: bold;
+  font-size: 12px;
+  text-align: center;
+}
+
+@media (min-width: 768px) and (max-width: 991px) {
+  .title {
+    font-size: 20px;
+  }
+
+  .fa.fa-bars {
+    display: none;
+  }
+
+  .topnav {
+    margin-top: 34px;
+    margin-right: 15px;
+  }
+  
+  .topnav a {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding: 10px;
+    display: initial;
+    font-size: 20px;
+  }
+
+  .topnav a:hover {
+    color:rgb(146, 146, 112);
+    cursor: pointer;
+  }
+
+  .big-news {
+    padding-bottom: 10px;
+    display: grid;
+    grid-template-columns: 2fr;
+    grid-template-rows: 150px 200px;
+    text-align: center;
+  }
+
+  .big-news-title {
+    padding-top: 80px;
+    margin-right: -20px;
+    margin-left: -20px;
+    width: 100%;
+    font-size: 18px;
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .big-news-text {
+    width: 100%;
+    grid-column: 2;
+    grid-row: 2;
+    margin-right: -20px;
+    margin-left: -20px;
+  }
+
+  .columbus {
+    max-width: 250px;
+    max-height: 300px;
+  }
+
+  .small-news {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: repeat(4, minmax(10px, auto));
+    grid-column-gap: 10px;
+    place-items: center;
+  }
+
+  .genghis, .dowland, .calendar, .couple {
+    max-width: 150px;
+  }
+
+  .genghis {
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .small-news-titles.a {
+    grid-column: 1;
+    grid-row: 2;
+  }
+
+  .small-news-text.a {
+    grid-column: 1;
+    grid-row: 3;
+  }
+
+  .calendar {
+    grid-column: 2;
+    grid-row: 1;
+  }  
+
+  .small-news-titles.b {
+    grid-column: 2;
+    grid-row: 2;
+  }
+
+  .small-news-text.b {
+    grid-column: 2;
+    grid-row: 3;
+  }
+
+  .dowland {
+    grid-column: 1;
+    grid-row: 4;
+  }
+
+  .small-news-titles.c {
+    grid-column: 1;
+    grid-row: 5;
+  }
+
+  .small-news-text.c {
+    grid-column: 1;
+    grid-row: 6;
+  }
+
+  .couple {
+    grid-column: 2;
+    grid-row: 4;
+  }
+
+  .small-news-titles.d {
+    grid-column: 2;
+    grid-row: 5;
+  }
+
+  .small-news-text.d {
+    grid-column: 2;
+    grid-row: 6;
+  }
+}
+
+@media (min-width: 992px) {
+  .logo {
+    max-width: 120px;
+  }
+
+  .title {
+    font-size: 25px;
+    margin-top: 40px;
+    align-self: flex-start;
+  }
+
+  .fa.fa-bars {
+    display: none;
+  }
+
+  .topnav {
+    align-self: center;
+    justify-content: space-between;
```

Comment:
When looking through the inspector this `justify-content: space-between;` does not know what to do because it needs a display :). "justify-content has no effect on this element since it’s neither a flex container nor a grid container."

Maybe try adding a 
```CSS
display: flex; or display: grid;
```